tests/testBaseStateMachine.py

```diff
@@ -164,8 +167,8 @@ def test5(self):
 
     def test6(self):
         """Test 6 for state machine"""
-        global test
-        test = 2
+        setTestTo2(test)    # Using the newly written function
```

Comment:
In this way doesn't work... the test variable passed as argument is not the variable declared at the start of the file.